axelrod/strategies/titfortat.py

```diff
@@ -642,10 +642,10 @@ def strategy(self, opponent: Player) -> Action:
 class Alexei(Player):
     """
     Plays similar to Tit-for-Tat, but always defect on last turn.
-    
+
     Names:
 
-    - Alexei's Strategy: [LessWrong2011]_    
+    - Alexei's Strategy: [LessWrong2011]_
```

Comment:
This is correct, it's the name for `Alexei`, it's just appearing in the diff because some white space is being cleared up on line 648.